lib/licenses.nix

```diff
@@ -505,6 +505,11 @@ lib.mapAttrs (n: v: v // { shortName = n; }) rec {
     free = false;
   };
 
+  nasa13 = spdx {
+    spdxId = "NASA-1.3";
+    fullName = "NASA Open Source Agreement 1.3";
+  };
```

Comment:
This should have `free = false`: 
https://www.gnu.org/licenses/license-list.html#NASA